pkg/cmd/roachtest/tests/failover.go

```diff
@@ -624,14 +712,20 @@ type blackholeFailer struct {
 	output bool
 }
 
-func (f *blackholeFailer) Setup(ctx context.Context)                    {}
-func (f *blackholeFailer) Ready(ctx context.Context, m cluster.Monitor) {}
+func (f *blackholeFailer) Setup(_ context.Context)                    {}
+func (f *blackholeFailer) Ready(_ context.Context, _ cluster.Monitor) {}
 
 func (f *blackholeFailer) Cleanup(ctx context.Context) {
+	if f.c.IsLocal() {
```

Comment:
We'll need to do this across all of the failers. We should also log a warning about it, so people don't start wondering why it isn't doing anything.